plugins/module_utils/network/dcnm/dcnm.py

```diff
@@ -83,8 +83,11 @@ def validate_list_of_dicts(param_list, spec, module=None):
                                                                                 spec[param].get('length_max')))
                 elif type == 'int':
                     item = v.check_type_int(item)
+                    min_value = 1
+                    if spec[param].get('range_min') is not None:
+                        min_value = spec[param].get('range_min')
                     if spec[param].get('range_max'):
-                        if 1 <= item <= spec[param].get('range_max'):
+                        if min_value <= item <= spec[param].get('range_max'):
```

Comment:
@mmudigon This change is causing a unit testcase for the `dcnm_interface` module to fail.  I did some investigation and discovered that it's because the `range_min` for the `ipv6_mask_len` parameter is `64` but one of the unit test cases uses a length of `48`. 

Is `48` a valid mask length?  

We need to either:

* Change the `range_min` for this parameter in the dcnm_interface module to something lower or...
* Change the test case to use a value above 48.

Reply:
If I update the fixture file to this the test passes:

```diff
diff --git a/tests/unit/modules/dcnm/fixtures/dcnm_intf_subint_configs.json b/tests/unit/modules/dcnm/fixtures/dcnm_intf_subint_configs.json
index 21c84fd..4f9cb91 100644
--- a/tests/unit/modules/dcnm/fixtures/dcnm_intf_subint_configs.json
+++ b/tests/unit/modules/dcnm/fixtures/dcnm_intf_subint_configs.json
@@ -172,7 +172,7 @@
                                "ipv6_addr": "fe80::02",
                                "int_vrf": "default",
                                "ipv4_mask_len": 16,
-                               "ipv6_mask_len": 48,
+                               "ipv6_mask_len": 68,
                                "fabric": "test_fabric",
                                "sno": "SAL1819SAN8",
                                "vlan": 201,
```

Reply:
I made the update in the PR to fix the failing test but we should discuss what the correct fix should be:

https://github.com/CiscoDevNet/ansible-dcnm/pull/124/commits/d53b1f9339af3af26d7ef33c85887ce43f957e78

Reply:
Though 48 is a valid IPv6 mask length, the minimum value suppported by DCNM is 64. So your fix is ok.